astropy/io/fits/scripts/fitsheader.py

```diff
@@ -42,14 +42,11 @@
 With Astropy installed, please run ``fitsheader --help`` to see the full usage
 documentation.
 """
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
```

Comment:
Is it okay to simply remove `unicode_literals`? It is used everywhere. There is even a special test marker to test both with and without `unicode_literals` in some cases (like `table`).